### DISCUSSION TITLE
container/auto-scaling

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#143] [General]
# container/auto-scaling

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/container/auto-scaling.html

**Comment by Everything1sPossible** [comment]
图 7-35 KADA 架构图 应是 图 7-35 KEDA 架构图

**Reply by isno** [reply]
谢谢，已经修正。

**Reply by zjyyzhengli** [reply]
截止目前，这个错误好像还并未修复。

**Reply by zjyyzhengli** [reply]
“图 7-39 KADA 架构图” -- 现在是这个编号

**Reply by isno** [reply]
感谢，已修复
